[2025-02-06T08:59:14.086+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T08:59:14.103+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.task_random_choice scheduled__2025-01-01T01:26:00+00:00 [queued]>
[2025-02-06T08:59:14.116+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.task_random_choice scheduled__2025-01-01T01:26:00+00:00 [queued]>
[2025-02-06T08:59:14.118+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-06T08:59:14.157+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): task_random_choice> on 2025-01-01 01:26:00+00:00
[2025-02-06T08:59:14.160+0000] {standard_task_runner.py:72} INFO - Started process 1252 to run task
[2025-02-06T08:59:14.162+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'my_first_dag', 'task_random_choice', 'scheduled__2025-01-01T01:26:00+00:00', '--job-id', '291', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp_40ht31c']
[2025-02-06T08:59:14.163+0000] {standard_task_runner.py:105} INFO - Job 291: Subtask task_random_choice
[2025-02-06T08:59:14.246+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.task_random_choice scheduled__2025-01-01T01:26:00+00:00 [running]> on host 128ac2960630
[2025-02-06T08:59:14.349+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='task_random_choice' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T01:26:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-01T01:26:00+00:00'
[2025-02-06T08:59:14.351+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-06T08:59:14.352+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-06T08:59:14.353+0000] {logging_mixin.py:190} INFO - Current task name:task_random_choice state:running start_date:2025-02-06 08:59:14.104321+00:00
[2025-02-06T08:59:14.353+0000] {logging_mixin.py:190} INFO - Dag name:my_first_dag and current dag run status:running
[2025-02-06T08:59:14.353+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T08:59:14.354+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-06T08:59:14.354+0000] {python.py:240} INFO - Done. Returned value was: task_2
[2025-02-06T08:59:14.355+0000] {branch.py:38} INFO - Branch into task_2
[2025-02-06T08:59:14.355+0000] {skipmixin.py:230} INFO - Following branch ('task_2',)
[2025-02-06T08:59:14.367+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-06T08:59:14.430+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T08:59:14.431+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=task_random_choice, run_id=scheduled__2025-01-01T01:26:00+00:00, execution_date=20250101T012600, start_date=20250206T085914, end_date=20250206T085914
[2025-02-06T08:59:14.462+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-06T08:59:14.463+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-06T08:59:14.464+0000] {logging_mixin.py:190} INFO - Dag name:my_first_dag queued_at:2025-02-06 08:59:10.454954+00:00
[2025-02-06T08:59:14.464+0000] {logging_mixin.py:190} INFO - Task hostname:128ac2960630 operator:BranchPythonOperator
[2025-02-06T08:59:14.496+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T08:59:14.527+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T08:59:14.539+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
